libs/interfaces/FileDetails.ts

```diff
@@ -86,19 +126,19 @@ export interface FileObject {
   /*
    * AITags field is populated only because the google-auto-tagging extension was executed synchronously and it received a successresponse.
    */
-  AITags?: object[];
+  AITags?: AITagItem[];
   /*
    * Field object which will contain the status of each extension at the time of completion of the update/upload request.
    */
   extensionStatus?: { [key: string]: string }
   /*
    * Consolidated embedded metadata associated with the file. It includes exif, iptc, and xmp data.
    */
-  embeddedMetadata?: object | null;
+  embeddedMetadata?: EmbeddedMetadataValues | null;
   /*
    * A key-value data associated with the asset. Before setting any custom metadata on an asset, you have to create the field using custom metadata fields API.
    */
-  customMetadata?: object;
+  customMetadata?: CmValues;
```

Comment:
Here M should be capital IMO. `CMValues`

Reply:
done